src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 :nth-child(-n + 1) {
+    background-image: url(../images/star-active.svg) !important;
+  }
+
+  &--2 :nth-child(-n + 2) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--3 :nth-child(-n + 3) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--5 :nth-child(-n + 5) {
```

Comment:
There is an extra space between '&--5' and ':nth-child(-n + 5)'. According to the code style rules, there should be no space between the element and the pseudo-class.